main/zoe/guide/contracts/loan.md

```diff
@@ -0,0 +1,143 @@
+# Loan
+
+<Zoe-Version/>
+
+##### [View the code on Github](https://github.com/Agoric/agoric-sdk/tree/master/packages/zoe/src/contracts/loan)
+##### [View all contracts on Github](https://github.com/Agoric/agoric-sdk/tree/master/packages/zoe/src/contracts)
+
+The basic loan contract allows a  borrower to add collateral of a
+particular brand and get a loan of another brand. The collateral (also
+known as margin) must be greater than the loan value, at an amount set
+by the Maintenance Margin Requirement (`mmr`) in the terms of the
+contract.
```

Comment:
This is a little confusing/unclear to me. I know that in stocks and similar, the margin requirement is usually set as a percentage, but this reads more like it's a set amount. I'd suggest rephrasing as whichever is appropriate:
"The collateral (also called *margin*) must be greater than the loan value. How much greater is specified by the
Maintenance Margin Requirement (`mmr`) in the contract's terms. The `mmr` value is an absolute value added to the loan value. For example, if the `mmr` is 1,000, and the loan value is 500, the collateral must be greater than 1500."

Or replace the last with "The `mmr` value is expressed as a minimum percentage of the collateral with respect to the loan value. So if the `mmr` is 25 and the loan value is 100, the collateral must be greater than 100 + (.25 x 100) or 125."   

Yes, there's at least one more way of doing this, depending on just what/how the `mmr` is expressed. My point is that as written, it could use more detail and an example as to how this contract is associating/using `mmr`, collateral, and loan value.

Reply:
Gotcha, ok thanks! Will revise